editor/actions.js

```diff
@@ -264,6 +264,30 @@ export function stopTyping() {
 }
 
 /**
+ * Returns an action object used in signalling that the user toggled the sidebar
+ *
+ * @return {Object}         Action object
+ */
+export function toggleSidebar() {
```

Comment:
I noticed that we have unit tests for a few existing action creators, should we cover those 2, too?

Reply:
Yeah! I know and I've written some of these, but honestly, I'm not sure these add any value for action creators just returning an action with the arguments as properties of the object.

Reply:
Yes, sometimes it's hard to justify the need for writing tests, but at the same time it's very easy to write them. It's your call, as I mentioned already. It's not a blocker.